spec/models/category_spec.rb

```diff
@@ -1,6 +1,5 @@
 require 'rails_helper'
```

Comment:
- [ ] As requested of you by the previous reviewer, when I run ```rspec``` to check the unit tests in your spec folder, they are all failing. Kindly look into why this is the case.

![image](https://user-images.githubusercontent.com/86318284/208608682-1e3a6341-b209-4f04-9968-315bb0932d66.png)